spec/s3_helper_spec.rb

```diff
@@ -8,12 +8,12 @@
     push
 
     pages_dir = Dir.glob('pages/*')
-    assert(pages_dir.empty?)
+    expect(pages_dir.empty?).to be(true)
```

Comment:
When learning rspec, one of the lessons is that you (usually) don't have to do `.to be(true)` or `.to be(false)`.
Instead `expect(pages_dir).to be_empty` (ruby will read `be_xxx` and try `xxx?`)

Read more here: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers